examples/v2_get_block_info.rs

```diff
@@ -39,5 +40,21 @@ async fn main() -> anyhow::Result<()> {
         println!("Last finalized {:#?}", ai);
     }
 
+    {
+        let identifier = AbsoluteBlockHeight::from(0);
+        let ai = client.get_block_info(identifier).await?;
+        println!("Block at absolute height {:?} {:#?}", identifier, ai);
+    }
+
+    {
+        let identifier = v2::RelativeBlockHeight {
+            genesis_index: 0.into(),
+            height:        0.into(),
+            restrict:      true,
+        };
+        let ai = client.get_block_info(identifier).await?;
+        println!("Block at relative height {:?} {:#?}", identifier, ai);
+    }
+
```

Comment:
Consider adding optional parameters to `App` for the absolute and relative height, such that it is easy to test whether it works for other values than 0. Especially when FFI is involved in the node ;)

Reply:
Yes please.